src/libc/posix_io/stat.rs

```diff
@@ -41,7 +41,7 @@ fn fstat(env: &mut Environment, fd: FileDescriptor, buf: MutVoidPtr) -> i32 {
     // TODO: error handling for unknown fd?
     let file = env.libc_state.posix_io.file_for_fd(fd).unwrap();
 
-    log!("Warning: fstat() call, this function is mostly unimplemented");
+    log_dbg!("Warning: fstat() call, this function is mostly unimplemented");
```

Comment:
How often does Wolf3D call fstat()? :p

Reply:
Hmm… I wish it were possible to somehow `assert` that `fstat()` is only used to read the filesize :(

Here's an idea: you could add a state variable for “have warned about fstat” so the log message is only shown once? I do something like that in `time.rs`.

Reply:
on the second thought, ~200 is not that much of logs, so i think i'll put log!() back

Reply:
Nobody actually reads them other than us devs anyway :p